### DISCUSSION TITLE
Disable mouse pad

### DISCUSSION THREAD

**Discussion by Shotman** [AppGround-io/bluetooth-keyboard-and-mouse-support#186] [Ideas]
Could it be possible to remove completly the mouse pad from the layout ? 
I'd like to use the phone as a button macro only keyboard and the mouse pad takes up a LOT of space :/

**Comment by zorazorr** [comment]
You can create custom layouts without mouse pad.

**Reply by Shotman** [reply]
I didn't see the option even though I have pruchased the pro version 

How can I completely remove the mouse pad ?

**Reply by Shotman** [reply]
Never mind I'm an idiot I was going into mouse/keyboard menu and not in custom profiles ...
